app/src/main/res/values-land/dimens.xml

```diff
@@ -300,13 +300,33 @@
   <dimen name="app_version_image_view_margin_start">36dp</dimen>
   <dimen name="app_version_text_view_margin_end">36dp</dimen>
 
-  <!-- Pin Password -->
-  <dimen name="pin_password_text_input_layout_width">244dp</dimen>
-  <item name="pin_password_constraintHorizontal_bias" format="float" type="integer">0.5</item>
-  <dimen name="pin_password_margin_start">72dp</dimen>
-  <dimen name="pin_password_margin_end">72dp</dimen>
-  <dimen name="pin_password_padding_top">48dp</dimen>
-  <dimen name="pin_password_min_width">496dp</dimen>
+  <!-- AdminPinActivity -->
+  <dimen name="admin_auth_activity_layout_width">0dp</dimen>
+  <dimen name="admin_auth_activity_margin_start">0dp</dimen>
+  <dimen name="admin_auth_activity_margin_end">0dp</dimen>
+  <dimen name="admin_auth_activity_padding_top">32dp</dimen>
+  <dimen name="admin_auth_activity_padding_bottom">104dp</dimen>
+  <dimen name="admin_auth_activity_admin_pin_main_text_view_margin_start">72dp</dimen>
+  <dimen name="admin_auth_activity_admin_pin_main_text_view_margin_end">72dp</dimen>
+  <dimen name="admin_auth_activity_input_width">304dp</dimen>
+
+  <!-- AdminPinActivity -->
+  <dimen name="admin_pin_activity_layout_width">@dimen/match_parent</dimen>
+  <dimen name="admin_pin_activity_margin_start">0dp</dimen>
+  <dimen name="admin_pin_activity_margin_end">0dp</dimen>
+  <dimen name="admin_pin_activity_padding_top">32dp</dimen>
+  <dimen name="admin_pin_activity_padding_bottom">104dp</dimen>
+  <dimen name="admin_pin_activity_admin_pin_main_text_view_margin_start">72dp</dimen>
+  <dimen name="admin_pin_activity_admin_pin_main_text_view_margin_end">72dp</dimen>
+  <dimen name="admin_pin_activity_input_width">304dp</dimen>
+
+  <!-- PinPasswordActivity -->
```

Comment:
Here, I think we can add  \<dimen name="pin_password_activity_layout_width"\>0dp\</dimen\> as we have set it in all other dimens.xml file for PinPasswordActivity. Though I have checked that even if we don't add this UI is still the same.

Reply:
Although its not required because in such cases the value from `value/dimens.xml` folder will be taken up. But still I have mentioned it now so as it make it look consistent and avoid confusion.

Thanks.

Reply:
Ok